tests/run-make/libtest-thread-limit/rmake.rs

```diff
@@ -0,0 +1,62 @@
+// libtest used to panic if it hit the thread limit. This often resulted in spurious test failures
+// (thread 'main' panicked at 'called Result::unwrap() on an Err value: Os
+// { code: 11, kind: WouldBlock, message: "Resource temporarily unavailable" }' ...
+// error: test failed, to rerun pass '--lib').
+// Since the fix in #81546, the test should continue to run synchronously
+// if it runs out of threads. Therefore, this test's final execution step
+// should succeed without an error.
+// See https://github.com/rust-lang/rust/pull/81546
+
+//@ only-linux
+// Reason: thread limit modification
```

Comment:
Suggestion: I would also dump a `ignore-cross-compile` directive here stating "this fails on `armhf-gnu` reasoons unknown" here.

Reply:
@rustbot author